Doppler.Integrations/Mapper/MapperSubscriber.cs

```diff
@@ -144,21 +145,27 @@ private void LogFieldsRejected(List<string> valuesList)
 
         public DopplerSubscriberDto TypeFormToSubscriberDTO(TypeformDTO rawSubscriber, ItemsDto allowedFields)
         {
-            DopplerSubscriberDto dopplerSubscriber = new DopplerSubscriberDto
-            {
-                Email = rawSubscriber
-                    .form_response
-                    .answers
-                    .FirstOrDefault(x => !String.IsNullOrEmpty(x.email))
-                    .email
-            };
-
-            if (String.IsNullOrEmpty(dopplerSubscriber.Email))
-            {
-                _log.LogWarning(String.Format("The response event: {0} to the form: {1} with ID: {2} has not included an email", rawSubscriber.event_id, rawSubscriber.form_response.definition.title, rawSubscriber.form_response.definition.id));
-                throw new ArgumentNullException(nameof(dopplerSubscriber.Email));
+            DopplerSubscriberDto dopplerSubscriber = new DopplerSubscriberDto();
+            try
+            {
+                dopplerSubscriber.Email = rawSubscriber
+                       .form_response
+                       .answers
+                       .FirstOrDefault(x => !String.IsNullOrEmpty(x.email))
+                       .email;
+            }
+            catch (Exception ex)
+            {
+                dopplerSubscriber.Email = DecodeDPLRID(rawSubscriber.form_response.hidden.dplrid);

```

Comment:
It would be nice doing this without using exceptions (exceptions are expensive), probably your exception is a null reference one, right?